roles/teleirc/handlers/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: install build dependencies
+  package:
+    state: present
+    name:
+      - gcc-c++
+      - libicu-devel
+
+- name: install runtime dependencies with yarn
+  become_user: teleirc
+  command: "yarn"
+  args:
+    chdir: "/usr/lib64/teleirc/{{ item.cn }}/"
+  with_items:
```

Comment:
You probably want `with_dict: "{{ bots }}"`? https://docs.ansible.com/ansible/2.4/playbooks_loops.html#looping-over-hashes

Reply:
@Qalthos This is exactly what I needed. You are awesome! :smile: Thanks for the pointer.